bin/check-credentials

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+set -o errexit
+
+die() {
+  stderr_echo "$@" && exit 1
+}
+
+stderr_echo() {
+  >&2 echo "$@"
+}
+
+check_heroku() {
+    endpoints="
+        travis-pro-web-staging
```

Comment:
maybe change this to `travis-pro-staging`? that way the arbitrary apps we check follow some sort of pattern.